java/src/org/certificatetransparency/ctlog/serialization/Deserializer.java

```diff
@@ -114,12 +115,12 @@ public static ParsedLogEntry parseLogEntry(InputStream merkleTreeLeaf, InputStre
 
     if (entryType == Ct.LogEntryType.X509_ENTRY) {
       Ct.X509ChainEntry x509EntryChain = parseX509ChainEntry(extraData,
-          treeLeaf.timestampedEntry.signedEntry.getX509());
+          treeLeaf.timestampedEntry.signedEntry.x509);
       logEntry.x509Entry = x509EntryChain;
     } else if (entryType == Ct.LogEntryType.PRECERT_ENTRY) {
       Ct.PrecertChainEntry preCertChain = parsePrecertChainEntry(extraData,
-          treeLeaf.timestampedEntry.signedEntry.getPrecert());
-       logEntry.precertEntry = preCertChain;
+          treeLeaf.timestampedEntry.signedEntry.preCert);
+      logEntry.precertEntry = preCertChain;
```

Comment:
it is to some extent an act of documentation.  here, preCertChain's type is clear for all to see.
folding those two lines into one, and it is left for the reader to go dig out (or hover with their mouse in their fancy IDE in Java land)